tests/comfy_cli/cql/test_default_workflow.py

```
@@ -182,3 +190,113 @@ def test_corrupt_bundle_surfaces_controlled_error(self, monkeypatch):
+        subfoldered = f"SD1.5/{DEFAULT_CHECKPOINT_NAME}"
+        oi = _object_info_with_checkpoints(["other.safetensors", subfoldered])
+        out, res = resolve_default_checkpoint(wf, oi)
+        assert out[CHECKPOINT_LOADER_ID]["inputs"]["ckpt_name"] == subfoldered
+        assert res.note is None
+        assert res.substituted_to is None
+        assert res.no_checkpoint is False
+
+    def test_object_info_without_checkpoint_node_fails_open(self):
+        # object_info present but no CheckpointLoaderSimple → can't tell → fail open.
+        wf = build_default_workflow(prompt="fox")
+        out, res = resolve_default_checkpoint(wf, {"KSampler": {"input": {"required": {}}}})
```

Pylint (4.0.6)

[warning] Unused variable 'out'

(W0612)